sm/plugins/tedge_apt_plugin/src/main.rs

```diff
@@ -37,6 +43,9 @@ pub enum PluginOp {
 pub enum InternalError {
     #[error("Fail to run `{cmd}`: {from}")]
     ExecError { cmd: String, from: std::io::Error },
+
+    #[error(transparent)]
+    InvalidCSV(#[from] csv::Error),
```

Comment:
I believe we agreed a couple of weeks ago to name errors which are `transparent from` as per below:
```suggestion
    FromCsv(#[from] csv::Error),
```